docs/en/observability/view-infrastructure-metrics.asciidoc

```diff
@@ -32,6 +32,7 @@ of your Kubernetes pods, grouped by namespace, and sorted by the memory usage va
 [role="screenshot"]
 image::images/kubernetes-filter.png[Kubernetes pod filtering]
 
+// lint ignore host1
 You can also use the search bar to create structured queries using {kibana-ref}/kuery-query.html[{kib} Query Language].
 For example, enter `host.hostname : "host1"`` to see only the information for `host1`.
```

Comment:
It looks like there's an extra backtick here. Removing the extra should allow the removal of the `// lint ignore host1`

```suggestion
For example, enter `host.hostname : "host1"` to see only the information for `host1`.
```

Reply:
Ah! I was wondering why that one was still throwing an error. 🦅 👁️